spec/models/rental_spec.rb

```diff
@@ -10,6 +10,28 @@
   it { is_expected.to validate_presence_of(:synced_id) }
   it { is_expected.to validate_uniqueness_of(:synced_id) }
 
+  describe '.connected' do
+    subject { described_class.connected }
+    let(:connection_1) { create(:connection) }
+    let(:rental_1) { connection_1.rental }
+    let(:connection_2) { create(:canceled_connection) }
+    let(:rental_2) { connection_2.rental }
+    let(:rental_3) { create(:rental, remote_rental: nil) }
+
+    it { is_expected.to eq [rental_1] }
+  end
+
+  describe '.not_connected' do
+    subject { described_class.not_connected }
+    let(:connection_1) { create(:connection) }
+    let(:rental_1) { connection_1.rental }
+    let(:connection_2) { create(:canceled_connection) }
+    let(:rental_2) { connection_2.rental }
+    let(:rental_3) { create(:rental, remote_rental: nil) }
+
+    it { is_expected.to eq [rental_2, rental_3] }
```

Comment:
As above for remote rental specs
1. Please use `match_array` instead of `eq`
2. Describe what spec returns
3. Optional - adding more expressive naming might be helpful to make it clearer